test/SecondaryRewarder/InvariantsSecondaryRewarder.t.sol

```diff
@@ -93,11 +95,12 @@ abstract contract InvariantsSecondaryRewarder is SecondaryRewarderSetupTest {
         amount = bound(amount, minAmount, maxAmount);
 
         vm.prank(account);
-        NOTIONAL.nTokenRedeem(account, CURRENCY_ID, uint96(amount), true, true);
+        // TODO: this will fail on an error "Residuals" if the amount is too large.
```

Comment:
@sbuljac This line is failing b/c we have refactored the nTokenRedeem method and you can no longer "accept residuals", that means if you are trying to redeem an amount that is too large this method will revert.